src/components/orders/OrderDetails/FillsTable.tsx

```diff
@@ -327,6 +327,10 @@ const FillsTable: React.FC<Props> = (props) => {
 
   const invertButton = <Icon icon={faExchangeAlt} onClick={invertPrice} />
 
+  const shownTrades = useMemo(() => {
```

Comment:
ube-mega-nit: It just came to mind cause i went to check the implementation after the name was not immediately clear, i find `tadesCurrentPage` more specific. Pls ignore this comment (thinking aloud)